doc/UsersGuide/source/solving.rst

```diff
@@ -327,6 +327,41 @@ system.
 
   body2.frame_a.r_0[1]
 
+Importing initial values from previous simulations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In many use cases it is useful to import initial values from previous simulations, possibly obtained with
+another Modelica tool, and saved in a .mat file. There are two different options to do that.
+
+The first option is to solve the initial equations specified by the Modelica model, using the previous simulation results to
+obtain good initial guesses for the iterative solvers. This can be very helpful in case the initialization problem involves the
+solution of large nonlinear systems of equations by means of iterative algorithms, whose convergence is sensitive to the selected
+initial guess. Importing a previously found solution allows the OpenModelica solver to pick very good initial guesses for the
+unknowns of the iterative solvers, thus achieving convergence with a few iterations at most. Since the initial equations
+are solved anyway, the values of all variables and derivatives, as well as of all parameters with `fixed = false` attribute,
+are re-computed and fully consistent with the selected initial conditions, even in case the previously saved simulation results
+refer to a slightly different model configuration. Note that parameters with `fixed = true` will also get their values from the
+imported .mat file, so if you want to change them you need to edit the .mat file accordingly. 
+
+This option is activated by selecting the simulation result file name in the OMEdit
+*Simulation Setup | Simulation Flag | Equation System Initialization File* input field, or by setting the additional simulation flag
+:ref:`-iif=<resultfile.mat> <simflag-iif>`. By activating the checkbox Save simulation flags inside the model, a custom annotation *__OpenModelica_simulationFlags(iif="filename.mat")* is added to the model, so this setting is saved with the model and is reused
```

Comment:
This link is not working in the generated PDF:
![image](https://user-images.githubusercontent.com/38031952/194837726-b27ca60b-3abb-4d7c-999e-f5107743b045.png)

I guess `<` and `>` are the problem. Maybe you need to escape them or remove them:

```suggestion
:ref:`-iif=\<resultfile.mat\> <simflag-iif>`. By activating the checkbox Save simulation flags inside the model, a custom annotation *__OpenModelica_simulationFlags(iif="filename.mat")* is added to the model, so this setting is saved with the model and is reused
```

Reply:
Oh and also use the full name of the check-box

![image](https://user-images.githubusercontent.com/38031952/194841488-30ba1b2c-f0b7-4b16-9e29-cc2a8dea2113.png)


```suggestion
:ref:`-iif=<resultfile.mat> <simflag-iif>`. By activating the checkbox *Save simulation flags inside the model i.e., __OpenModelica_simulationFlags annotation*, a custom annotation *__OpenModelica_simulationFlags(iif="filename.mat")* is added to the model, so this setting is saved with the model and is reused
```